packages/calcite-components/src/components/handle/handle.scss

```diff
@@ -1,5 +1,34 @@
+/**
+* CSS Custom Properties  
+*
+* These properties can be overridden using the component's tag as selector.
+*
+* @prop --calcite-handle-text-color: Specifies the text color of the component.
+* @prop --calcite-handle-text-color-hover: Specifies the text color of the component when in hover state.
+* @prop --calcite-handle-text-color-focus: Specifies the text color of the component when in focus state.
+* @prop --calcite-handle-text-color-selected: Specifies the text color of the component when selected.
+
+* @prop --calcite-handle-background-color: Specifies the background color of the component.
+* @prop --calcite-handle-background-color-hover: Specifies the background color of the component when in hover state.
+* @prop --calcite-handle-background-color-selected: Specifies the background color of the component when selected.
+*/
+
 :host {
   @apply flex;
+
+  --calcite-handle-text-color: var(--calcite-color-border-input);
+  --calcite-handle-background-color: transparent;
+
+  /* hover */
```

Comment:
Nitpick: I don't think we need these comments if we group props accordingly. We should add a note to the workflow doc about this. cc @alisonailea